src/hmem_cuda.c

```diff
@@ -606,8 +573,7 @@ int cuda_hmem_init(void)
 	 * - cudaMemcpy() is available
 	 */
 	cuda_attr.use_ipc =
-		(cuda_attr.p2p_access_supported || cuda_attr.device_count == 1) &&
-		(cuda_attr.xfer_setting != CUDA_XFER_DISABLED);
+		cuda_attr.p2p_access_supported || cuda_attr.device_count == 1;
```

Comment:
is `p2p_access_supported` already true when `cuda_attr.device_count == 1` ?

Reply:
No, `p2p_access_supported` is initialized to `false` and the P2P detection function returns early if the number of CUDA devices detected is $\leqslant 1$